### ISSUE TITLE
didSelect delegate function seems broken !?

### ISSUE THREAD

**Issue by undeaDD** [yervandsar/ExpandedTabBar#15]
didSelect only gets called for items inside the more container?
the other tabBarItems outside get ignored :S 

this cant even be circumvented by conforming to UITabBarControllerDelegate ( redundant conformance to UITabBarControllerDelegate )

( tested on latest version & iphone 11 )
really like v3.0.0 otherwise ^^ great job

**Comment by yervandsar** [comment]
@undeaDD if you see this method in ExpandedDelegation
```swift
    func expandedTabBarController(_ tabBarController: UITabBarController,
                                  didSelect viewController: UIViewController,
                                  withItem tabBarItem: UITabBarItem?) {
        // Do some logic here
    }
```

You can get other select actions with overriding parent's implementation like this in your view controller.

```swift
    override func tabBarController(_ tabBarController: UITabBarController,
                                                        didSelect viewController: UIViewController) {
        super.tabBarController(tabBarController, didSelect: viewController)
        // Your logic
    }
```
**NOTE:** `super.tabBarController(tabBarController, didSelect: viewController)` is **REQIURED**

*BTW* it will added as soon as posible.

**Comment by undeaDD** [comment]
[
<img width="738" alt="Screenshot 2020-06-20 at 22 26 16" src="https://user-images.githubusercontent.com/8116188/85210986-2601f800-b345-11ea-8b2e-2dafeaac85e7.png">
](url)

**Comment by undeaDD** [comment]
**another thing that came to my attention:**
_"parentContainerView"_ inside:
_"internal func refreshContainerPosition(with size: CGSize)"_
is nil and crashes the app when less than 6 viewcontrollers are assigned

but that could easily be my fault.
i use a hybrid approach (storyboard + programmatically viewcontroller assigning)

<img width="755" alt="Screenshot 2020-06-20 at 22 46 53" src="https://user-images.githubusercontent.com/8116188/85211286-f2749d00-b347-11ea-86ef-72f421b3b8a3.png">

calling the dedicated setup function wont help here either because there may be less than 6 viewcontrollers ( dynamically assigned at runtime )

<img width="499" alt="Screenshot 2020-06-20 at 22 47 33" src="https://user-images.githubusercontent.com/8116188/85211295-0ae4b780-b348-11ea-866b-75e089c8b9c2.png">

**Comment by yervandsar** [comment]
@undeaDD Fixed in [v3.0.1](https://github.com/yervandsar/ExpandedTabBar/pull/16)
